src/AppBridgeBlock.spec.ts

```diff
@@ -381,10 +381,10 @@ describe('AppBridgeBlockTest', () => {
     test('updateColor with default language', () => {
         setTranslationLanguage('');
 
-        const frontifyColor = FrontifyColorDummy.red();
+        const frontifyColor = ColorDummy.red();
```

Comment:
Constant name as well: 
```suggestion
        const color = ColorDummy.red();
```